lib/tasks/reminder.rake

```diff
@@ -0,0 +1,21 @@
+namespace :reminder do
+  desc "Send actionable approver reminder email for pending requests"
+  task actionable_approvers: :environment do
+    props = Proposal.pending.where(client_data_type: "Ncr::WorkOrder")
+    props.each do |proposal|
+      send_reminder_email(proposal)
+    end
+  end
+
+  private
+
+  def send_reminder_email(proposal)
+    user = proposal.client_data.current_approver or return
```

Comment:
`proposal.client_data.current_approver or return` -- not sure that is super readable to me. What about wrapping this all in conditional to take care of the situation where there is not an approver?